modules/tinymce/src/core/main/ts/fmt/ExpandRange.ts

```diff
@@ -18,6 +18,7 @@ import { isEmpty } from '../dom/Empty';
 import { Format } from '../api/fmt/Format';
 import * as FormatUtils from './FormatUtils';
 
+const onlyWhitespaces = / +/;
```

Comment:
I do wonder if we have a util function somewhere else for this already... Though we probably check for multiple kinds of whitespace, and I suppose we only want to test for spaces here?

Reply:
They don't check for this, and I'm not confident enough in what knock-on effects it might have to change it.

Reply:
The name of this regex is very misleading, as this is only checking if it **spaces** not if it's whitespace.